aat/src/resources/features/F-023/S-007.td.json

```diff
@@ -0,0 +1,33 @@
+{
+  "title": "Must return a negative response on non existent Case Type ID",
+  "_guid_": "S-007",
+  "_extends_": "F-023_Test_Data_Base",
+  "specs": [
+    "an active profile in CCD - wants to see a list of valid documents they can print",
+    "contains a non existent caseTypeID"
+  ],
+
+  "request": {
+    "pathVariables": {
+      "ctid": "AAT_AUTH_00",
+      "cid": "1111222233334444"
+    }
+    },
+
+  "expectedResponse": {
+    "responseCode": 404,
+    "responseMessage": "Not Found",
+
```

Comment:
Please extent the common 400 response object. That will bring the 2 lines here and the headers below.

Reply:
done